walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -15,9 +15,118 @@
 package com.google.sps;
 
 import java.util.Collection;
+import java.util.ArrayList;
+import java.util.Iterator;
 
 public final class FindMeetingQuery {
```

Comment:
I would add a javadoc to this class. Your diagrams in the test file would be nice in this file as well!

https://google.github.io/styleguide/javaguide.html#s7-javadoc

Reply:
Is a javadoc basically just lines of comments explaining what happens?

Reply:
Yeah, they use `/* ... */` style instead of `// ...` (which are implementation comments) and they're intended to be read by the person using the class. So, if someone wanted to use the public class FindMeetingQuery, they would understand how to use it just from reading the javadoc, and they wouldn't have to read any of the code.

Reply:
Ok, cool! That makes sense!

Reply:
I just added the javadoc for the class!